packages/dcos-history/extra/history/server.py

```diff
@@ -1,93 +1,8 @@
-import logging
 import os
-import sys
+import history.server_util
```

Comment:
What is the motivation for splitting out all the business logic into 'server_util' if the resulting server module is almost completely trivial?